pkg/cli/common.go

```diff
@@ -270,6 +271,7 @@ func GetBudFlags(flags *BudResults) pflag.FlagSet {
 	fs.BoolVar(&flags.Stdin, "stdin", false, "pass stdin into containers")
 	fs.StringArrayVarP(&flags.Tag, "tag", "t", []string{}, "tagged `name` to apply to the built image")
 	fs.StringVarP(&flags.BuildOutput, "output", "o", "", "output destination (format: type=local,dest=path)")
+	fs.BoolVar(&flags.Push, "push", false, "Shorthand for `--output=type=registry`")
```

Comment:
I thought we decided we were not going to support the push flag in buildah but only in podman build?

Reply:
Yes, you're right, this was added in order to test the changes as part of this draft PR.